Kconfig.nrfxlib

```diff
@@ -16,4 +16,5 @@ rsource "nrf_rpc/Kconfig"
 rsource "zboss/Kconfig"
 rsource "openthread/Kconfig"
 
```

Comment:
This empty line should rather be below `rsource "nrf_802154_sl/Kconfig"` I believe.